packages/wallet-backend/src/index.ts

```diff
@@ -23,7 +23,15 @@ app.disable('X-Powered-By')
 
 app.use(
   cors({
-    origin: 'http://localhost:4003',
+    origin: [
+      'http://localhost:4003',
```

Comment:
should these be in env file?

Reply:
Well, I think as the domain will be fixed, it can be here as well.